Composer/packages/client/src/pages/design/exportSkillModal/styles.ts

```diff
@@ -32,11 +32,11 @@ export const styles = {
 
     label: ContentContainer;
   `,
-  content: css`
-    flex: 1;
+  content: {
+    flex: 1,
```

Comment:
isn't display: flex required here before u set flex:1?

Reply:
display: flex set in the container css, content is the child element